README.md

```diff
@@ -27,3 +26,17 @@ about it at the start of each recursive call.
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. You do not need to prove that the invariant is correct. Add
 your answer to this markdown file.
+
+
+## Invariant Answer
+At the beginning of the function, a good invariant for my implementation should be s-1, "At this point, the array contains the each element of the Fibonacci Sequence up to value s-1."
+
+At the end of recursive of fib(), just before returning the array, the invariant is s, "At this stage, the array includes every element of the Fibonacci Sequence up to value s."
```

Comment:
Ok, please go back to the slides and have a look how we used invariants. You can't just give me two.